src/nevermined/api/nfts/NFT1155Api.ts

```diff
@@ -78,72 +80,55 @@ export class NFT1155Api extends NFTsBaseApi {
     }
 
 
+
     /**
-     * Create a new NFT Nevermined NFT.
+     * Creates a new Nevermined asset associted to a NFT (ERC-1155).
      *
      * @example
      * ```ts
-     * ddo = await nevermined.nfts1155.create(
+     * const assetAttributes = AssetAttributes.getInstance({
      *           metadata,
-     *           artist,
-     *           BigNumber.from(10),
+     *           price: assetRewards,
+     *           serviceTypes: ['nft-sales', 'nft-access']
+     *       })
+     * const nftAttributes = NFTAttributes.getNFT1155Instance({                
+     *           nftContractAddress: nftUpgradeable.address,
+     *           cap: cappedAmount,
+     *           amount: numberNFTs,
      *           royaltyAttributes,
-     *           new AssetRewards(),
-     *           BigNumber.from(1),
-     *           token.getAddress()
+     *           preMint
+     *       })            
+     * const ddo = await nevermined.nfts1155.create(
+     *           assetAttributes,
+     *           nftAttributes,
+     *           publisher
      *       )
      * ```
      *
-     * @param metadata - The metadata associated with the NFT.
-     * @param publisher -The account of the creator od the NFT.
-     * @param cap - The max number of nfts.
-     * @param royaltyAttributes - The royalties associated with the NFT.
-     * @param assetRewards - The sales reward distribution.
-     * @param nftAmount - The amount of NFTs that an address needs to hold in order to access the DID's protected assets. Leave it undefined and it will default to 1.
-     * @param erc20TokenAddress - The ERC-20 Token used to price the NFT.
-     * @param preMint - Set to true to mint _nftAmount_ during creation.
-     * @param nftMetadata - Url to the NFT metadata.
-     * @param appId - The id of the application creating the NFT.
+     * @param assetAttributes - Attributes describing the asset
+     * @param nftAttributes -Attributes describing the NFT (ERC-721) associated to the asset
+     * @param publisher - The account publishing the asset
      * @param publishMetadata - Allows to specify if the metadata should be stored in different backends
      * @param txParams - Optional transaction parameters
+     *
      * @returns The newly registered {@link DDO}.
-     */
-    public create(
-        metadata: MetaData,
+     */    
+     public create(
+        assetAttributes: AssetAttributes,
+        nftAttributes: NFTAttributes,        
```

Comment:
I think NftAttributes should extend AssetAttributes instead of having to use two parameters

Reply:
done